packages/core/src/types/provider.ts

```diff
@@ -405,3 +407,18 @@ export type PlaybackInfo = {
     }[];
   };
 };
+
+export type GetMetricsArgs = {
```

Comment:
@victorges when you get a chance, can you give me more info on what queries are currently supported by the viewership API? Are time binning and others supported yet?